evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -27,6 +27,11 @@ contract Operator is
     uint8 dataVersion;
   }
 
+  struct Authorization {
+    bool authorized;
+    uint256 index;
```

Comment:
You can save a word of storage and a bunch of gas by making this a `uint255`, but realistically it probably doesn't need to be any bigger than a `uint8`.